### ISSUE TITLE
EXR saved form Vray Settings does not match EXR saving from VFB

### ISSUE THREAD

**Issue by forzasila** [usakhelo/batchcam#10]
Hello,

I noticed differences between EXRs saved using the script and EXR saved from VFB.
I'm using 3D Max 2024.2.1 and Vray 6 update 2.1 saving EXR with all passes in and opening it with PS and Exr-io plugin.

I had no problem until I updated Vray to this last version.
I wonder if this is due to the new Color management system in Max.

Hope you can update and fix it.

Many thanks!

**Comment by forzasila** [comment]
Hello,

I did extra tests and it's like batchcam is saving EXR without VFB Display Correction.

Thanks,
Alex

**Comment by usakhelo** [comment]
Hello, sorry for the delay. I had to look into my original notes and it looks like Vray VFB was supported at some point. Probably Vray has changed something and now script needs to be adjusted. Thanks for checking in the issue. I'll try to fix this.

**Comment by usakhelo** [comment]
Can't reproduce this issue. In my tests with 2025 + latest Vray script saves the output with all Vray VFG changes. Could you provide a small test scene, and the output that shows the difference?

**Comment by forzasila** [comment]
I'm using 3DMax 2024.2.1 update and the latest Vray. I work with big scene but I could simply it for you. How can I send it? Wetransfer?

**Comment by usakhelo** [comment]
Attach the files or the zip right here in your reply.
Please keep the scene as simple as possible, like a teapot or something, but make sure the camera has necessary BatchCam properties set. And I will need two exrs that show the difference.

**Comment by forzasila** [comment]
I had to zip the files

**Comment by forzasila** [comment]
And the Outputs, with basic VFB color correction.
Test_Scene_01_vray_exr is the EXR output, opened with PS and resaved as JPG,
Can't load EXR file as it's too big.
 Let me now if it makes sense pls.
[Test renders.zip](https://github.com/user-attachments/files/15776029/Test.renders.zip)


Thanks

**Comment by usakhelo** [comment]
I haven't tried the scene yet, but original issue was about differences between batchcam and vray-vfb EXRs, right? Not the difference between EXR and JPG.

**Comment by forzasila** [comment]
Yes, correct. EXR file saved using batchcam are different from EXR saved from VrayVFB with colors adjustment.
No problems on JPGs using batchcam.
